### ISSUE TITLE
Can't disable NuGet publishing

### ISSUE THREAD

**Issue by rgwood** [xoofx/dotnet-releaser#31]
Hi, I'm running into an odd issue using [this `dotnet-releaser.toml` file](https://github.com/rgwood/RezoningScraper/blob/master/dotnet-releaser.toml).

I want to publish to a GitHub release and I do not want to publish to NuGet, so I've added this to my config:

```
[nuget]
publish = false
```

However, `dotnet-releaser publish` fails because it wants a NuGet token:

```
❯ dotnet-releaser publish --force dotnet-releaser.toml --github-token=REDACTED
2022/02/27 09:59:50.849 info: [1] dotnet-releaser 0.3.2 - publish
── Configuring ─────────────────────────────────────────────────────────────────────────
2022/02/27 09:59:50.922 info: [2] Loading configuration from C:\Users\reill\source\RezoningScraper\dotnet-releaser.toml
2022/02/27 09:59:50.992 info: [3] Loading 1 projects
2022/02/27 09:59:52.972 info: [4] Packages and Projects
┌─────────────────┬──────┬─────────┬────────────────────┬───────────┬───────────┬───────┐
│ Project         │ Kind │ Version │ TargetFramework(s) │ License   │ Packable? │ Test? │
├─────────────────┼──────┼─────────┼────────────────────┼───────────┼───────────┼───────┤
│ …               │      │         │                    │           │           │       │
│ RezoningScraper │ exe  │ 2.3.1.0 │ net6.0             │ Unlicense │     x     │       │
└─────────────────┴──────┴─────────┴────────────────────┴───────────┴───────────┴───────┘
2022/02/27 09:59:53.010 info: [5] Connecting to GitHub
2022/02/27 09:59:53.499 fail: [6] Publishing to NuGet requires to pass --nuget-token
```

If I give it a fake NuGet token it gets a bit farther and then fails later on:

```
── Publishing Packages - version: 2.3.1.0 ─────────────────────────────────────────────────────────────────────────
2022/02/27 09:50:22.952 info: [198] Publishing NuGet 2.3.1.0
2022/02/27 09:50:23.492 fail: [199] error: File does not exist (*.nupkg).
```

Not sure why it's trying to publish to NuGet - have I missed something obvious? I took a quick look at the source code and [it seems like dotnet-releaser always expects a NuGet token](https://github.com/xoofx/dotnet-releaser/blob/dbfb8be95e505df6317894ffbea90d7bc61d66d2/src/dotnet-releaser/ReleaserApp.Configuring.cs#L136-L140).

## Version Info

- dotnet-releaser v0.3.2
- .NET SDK v6.0.2
- Windows 11

**Comment by xoofx** [comment]
Thanks, good catch! Fixed by commit c4fec2f and 9000623. Will be available in `0.3.3+`

**Comment by rgwood** [comment]
Awesome, thanks for another super speedy fix!
